Description: new certificate meeting OpenSSL SECLEVEL 2
 Fix compilation error "ca md too weak" by generating a new certificate meeting
 the requirements of OpenSSL SECLEVEL 2.
 .
 See https://wiki.debian.org/ContinuousIntegration/TriagingTips/openssl-1.1.1
 .
Author: Emanuele Rocca <ema@debian.org>
Bug-Debian: https://bugs.debian.org/911836
Last-Update: 2019-02-25

--- pytest-localserver-0.5.0.orig/pytest_localserver/server.pem
+++ pytest-localserver-0.5.0/pytest_localserver/server.pem
@@ -1,81 +1,49 @@
------BEGIN RSA PRIVATE KEY-----
-MIIJKAIBAAKCAgEA0phLUETPqoTk32D857Cg1HHtpp07IR34uz5/Vnw+5HNG+51u
-LQCrRktVySyfDGhJTVTv+Ysx3+g4Fq0uU/sYvaZSoZSziH7LetoWpYuU34QAMFTg
-8Pk7Tli1axUJz0G5pO6ClUf9i25pFS8/lbk335d4MGc6w//+pthpuhzan8PaufP1
-DOVQPIcvrsDjZI1/yb0zBKTYKPaRGoTrM2lgX3BKVApSAq1fL8NYzBx9vR3Fwo/5
-sRPRWl9BpQNxv6zDYV2CtrfK9D1opxZulTi45r0aPcVqY1tY79ZdXoO+0dVVJSnE
-WH3RmCDKS229Sg6wTZp+qIHw8WTkAXxX7Wg7wHp5l7Ml7gnoM00pzEvk1ZwPnzyX
-ai0R3ADhhmJx3nDUNx+3QmQ+miFD1brn1W+oUf//Wylx2zP5numWVrXuwvPoHUez
-dJwHXpIAYxByG8RhER/OB7J45lFgwQPEYpkATP3LMZNc19K6jqp70HTgW+3ICTWC
-IX+5zyOGDLPJnxmmGM5pNuy/s5k4Ojf7+k39fxGwUEZvQ2otlj7lKddBYwjcyQeg
-Q9d25ubiHPGaFlAJG1PdUPumulsPto3Kf74d88SXDIjXq9DemVETlUk/hCB0ohNx
-r3tKFNNS0rOd5VPzqPowbIszJmnPQ7FWNFm8BVNE7bIMq0e3KFhfqB/SgCsCAwEA
-AQKCAgBEpojGnefkenXxq7hF/vouxwTlr5O68/5Fa9Yu50nJRxr4oxVrbjBnNd+9
-OOKQNN7QasPf4VRc3WKLYeEcV1p1WGbPbZ4o/MxwO+/t/1aElo9JMiIx6809eQWK
-5szP12khLu86osEwVsnCsihUMpDYPpRbkwtBIY5t7VZp6UYLltkSjUhw77/8O0V7
-/j0iZk4mAZtEBEjC8b3Tm0jg9ZUJB53zT3n0jXfeJhwoTZFfuPjit/R5mTBB+ZN4
-q2rFRRhcARO/ZKKwg+Q9s75QvrDn5vVK9bP8t6aCaKr/SItH8/dRqg/rmPheUzIe
-7KMsJSWr5pUrcVCOIImDQnmXVsdDG6k6JLZkWxrE3jXC1leruX3rUXUfKrH7lBcg
-tSRWGGtc3fIittYrKCQ99XA27nhrWdAV9Qh0zNPVE9Agu+K2ZmDhaaYgykrGQsl/
-LRBfDidyEbo7psbTFzKrRfrwEUUyxjm6/megnQdE3s47/bh4uxRT6kxYFeW69NLw
-SQzlCs/HhFPUTbFSPD8gJrIOoXbjphTLOBDTgVWwN0WEkoLMw+0dJXyRGPmuwirQ
-m4qjNqqFS9LTjYhQsnNWJxiSgotclbAwAJLJAN7lGTv3pBlj+qvBWugqSNS5zxhh
-badWaWSo13tWIhLESwdl0gcXz75MnQUf7mqUN+r1LLmdxRSHwQKCAQEA/irZDbij
-ZU6TdOmgBb0GnsrvKBUNwLyJPIBQ4WPeNlbSunc9gM/oxdcAi4JS0SxyMBidRkPh
-+fEFV7ai8nokMJWYpNoNu6BJn3/YZjoOi75P5LHHBLByCqOHNMKXIsucscs7ATxy
-oGjCW9uKrgvk9u66kE4yXCyYz+ALbC7358XJPOmiKkHgPufiPPOoUCZZE3H2wfxJ
-tWJpuitCqspRQmqNTZzQ2tSrzQNdBpIPAUbwDucWVnZNO2kuZYH/Op2vTPZWbvyE
-K5hIE0oDvFzIa6k1EaM7qwHpnjnDYa5sIgRgXOizFGWuI5NLaOT0hnykkXNE8B3b
-FqzE2ZrsyjqbowKCAQEA1B0EvwQFpvWOe2BrA72qEYYCGQJs6eYk0wOjT6AOW+2h
-osHz0DOR4vtCQZWUqCx6gcfH8IYAQda6jPhtl5rwCoI2SVHGisFIEi/+b46Ellvo
-86Q3dsbBFeyZnXe6iiFGqeQNevHFJZI+Hl7D4kn2Xti+fCu1mD34zPB6BK2b1rOQ
-LZ4nUJZ2ihW4G8p5KDF8NaWHneZmDXOuWjDGJhkJi9oWTyeO6YYZNy9ibJkhgqWm
-hSPd8TWfwPAx2iu5Xtv4VxLEYBb0ODuOo7lbe7bHi2Kl6tRJJJrFNlFfY3gThkGy
-x7JymuJKn1eFIqlnC/kCxbp/QTUxRSIyg26qYeJR2QKCAQEAgXWHqxKBv2cfemxJ
-AsM+LQGGgS8AG0ysxjFBgRwiIiFlIm7717GOib6yW1zdHAf8fXz8Fr8ayfk21G+F
-XeZSqqB36HfciNXiyigPLDqkEA+2l/DpQv0wiTSz+G6pFqMJRBIVEtMcO5JAdRNE
-9tO+IvyD8F60Dfj1OOSQTOE7ikVW8/y4ibeWJdRsisk80N5Hfr0Mh9AeJxJicd35
-Dg8RBrhJEuAuDVhHzZvJ9T/N/sOIIL8ZxJ5rRjiT1kY5nzyT/qp2af+avq3JAHIx
-pu7HcvbzvfEbwkWuHdakKwvxrf32qzwngtsrsJkkzt/XoOy50hAZbAh5AHqtpXzg
-4Tyq6wKCAQAyy1gtnfr0hm0+YlJ+LNcjJyItdwU55eD/nylmkf+eSAOjBCssuoy+
-/EhbiER3GV+k4ICiupQpMBCwXXVxDqtY+OGjITAYXjyyDkgx0sExeIUKg1K84M+w
-Pl5y9Q74sQtI99G6lVcOrjyt7SefcvLKt8C134m3EXAZM6UkpaZh/dIS+oKTq9Rr
-MNRL42qsE12ZQzB8wYXCxucDthZdXTWKBgm9ns18Yp57Np+tPeZmcOC1lWc6sgq1
-m0903W+gpbCMuXPJRvXo2WpJDsQ3lgem+1KcL4XsfBup7EaZfVG/ns//Pl5vdK1G
-ByFcsZB5r4HYc9axeNl5orzR+JhYpGfRAoIBAG5Bf3D91R/sXUpX2CEpU1/UURcD
-F8fikMV6WdV+n5DNLNb2m5/yzgOLRxfW3Q9ys9a6zbC9/lqxggZpYSAhW9MzRFq+
-ObWTLgw/Be1pJG1U6JP+O7iZlVPKV1GNyMFp/JHpP7R8CPjc54oFH52RtbJXlErv
-hRaHfQ3r2+vuHjzdLWkJTlRZNbGvTxnUn7qiX8EgL2ZQLDMe/VtoyTJZo/EprjPK
-8jD6rwN3OGvxfFuze8zr1G69tKy2rRG3KYtvS4SADdaeUYx6XwScTvjS7b7X7Ca4
-8528O88TBB4OV5fGAaYeglo0fqFyLUj5C4aQAFtBJJbWCECrVLYqjIyXxm4=
------END RSA PRIVATE KEY-----
+-----BEGIN PRIVATE KEY-----
+MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQCd3M8rihBCV9so
+UdRjbhIl32y2+O9Gufxi4slIiANayDaIHJDSqpxgrGGV1IZpyr4Whyq77x8zsSg0
+cFSZemqeHVJVr9vQZTcIl+o84ktMVtv8CShLgNzf0iwa+2q9t7/9w91c9PG2tWS7
+bOGC/6RRprxIj8icQe6M9Sh3jWBR5VTzlb1Xczf6WMq+9OPJf+xGdq67SaMdWxd9
+6NQ+F2ZAbGB42agZWlvsdIgY/PllfSjkccUteiUk6y/npbtyel1rgW/WSqzwmjgK
+KOEUGvfyzXZiv3f93ZGesxUppfeuh3cQSvBLMfDx3VQY82amX/9uOEUnEtmMeYqb
+CInQUSszAgMBAAECggEAWMs94HCdKqP+StSUUYU1QgZFQToOZMrycrC7pUeUE3kw
+aykeMND0etAbZ8qN/G2ktEk9OU6MT2gjhDN28olP9TXq/PWY/8Qey3+fqrViw0LM
+JB0GhEmsc1zloBuw85uqq1ODbOj+GNY5Hc9lnWisBZ1ypKHzpijTxIeVRoHIHues
+0tmaATHpCxdd2zAUpSanpSBO5WGJ2/PZ+DvethwwZTUGWctQ38MMuIJG7C+LzS0Q
+0QRPTrmw0BR+WfX7WpataUm8z1FxNrIFU4ie4dQ5l4tyJk+VeVzYEZ1ggs7nqglA
+gRNuYX8sz2cv68kf5+ELl9V9+QaWjs+iPXpO0iRhEQKBgQDK0U8/qVEpOk6ovcOX
+wIEeElsXh5RyoWbxA8xr8wHteI18uNy9f3QCrg1MURwEKzzeJdqoIB4HRMbkcMjq
+Ab2g2rvPG0nyw4Ktgwj/AGTlJpPo76ki81c4yxXxRS7Bvr6A6qUmt1sVIPAKeGja
+3Yh0geSopoHrVFzoQ8nUNM/RmQKBgQDHQcQtCJnytvQB8cMUdKRaYeo20dAud4Lb
+KW+wFIG22ycqXwW5eWuXiIJrM9RuDVypLVJTG+UcF6Xp0dIwzpuOHPSPGsgOK6cd
+Q5os2orRv7z0PR9R7idcu7bZLkjC/xdwZenndfPilUeLdJoo4tCy7NGvf8cKAHvo
+PqSs5U+6qwKBgFf5IGHhxJWYeAQTg+KwGpF7rs1u/sIWJnvGhQYR+zURRHXkDcOY
+pC6BZr1CtV9NjfJiAw1IGR5YA9mU3mE6bhF2r14vXQ0TJvyySNWrSxhmemPgiMHj
+QWMeHB9yPczGOwXaQdagU76/cb9o6Tp+OzEmIpPQ3avBBSeuN753RemJAoGBALHl
+Lo+CQWfoHZvqy99kcCPdjW0aD4Kc4rBmozeSZ2jo6mOy6Hp6RCvLDZ1Gbd8/gL3u
+H7Thj8BwAInr8XvvygOsKE/nejrkkFPZOjnXd97iD4IDXAYa2Pu6//mQkBfABZ4s
+7xLeRTrDSlfgZ44fSnKkamUEcKCmY/mqi/1s0az3AoGAc0xeh+fqV3sv5giBxbnB
+uBOu4Ip49xoYkiVyftl+pIDV4fXraSzhq+bAmMmlnD1LnirkTVYER0UnWed4kNlW
+4AL7qY3a3DyCnfE+ONU3kcqpYS7tRBMC70uR3LWO4AB2QmQeYHOUaYNflQdYLq7w
+u5ml0SPCWn1JW61f+n6fDBg=
+-----END PRIVATE KEY-----
 -----BEGIN CERTIFICATE-----
-MIIFODCCAyACAQEwDQYJKoZIhvcNAQEFBQAwezEbMBkGA1UEChMScHl0ZXN0LWxv
-Y2Fsc2VydmVyMSQwIgYDVQQLExtDZXJ0aWZpY2F0ZSBBdXRob3JpdHkgRGVwdC4x
-FTATBgNVBAMTDGxvY2FsaG9zdCBDQTEfMB0GCSqGSIb3DQEJARYQYmFzdGlAcmVk
-dG9hZC5kZTAeFw0xMTA1MDcxNjEzMzFaFw0xMTA2MDYxNjEzMzFaMEkxGzAZBgNV
-BAoTEnB5dGVzdC1sb2NhbHNlcnZlcjEWMBQGA1UECxMNVGVzdGluZyBEZXB0LjES
-MBAGA1UEAxMJMTI3LjAuMC4xMIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKC
-AgEA0phLUETPqoTk32D857Cg1HHtpp07IR34uz5/Vnw+5HNG+51uLQCrRktVySyf
-DGhJTVTv+Ysx3+g4Fq0uU/sYvaZSoZSziH7LetoWpYuU34QAMFTg8Pk7Tli1axUJ
-z0G5pO6ClUf9i25pFS8/lbk335d4MGc6w//+pthpuhzan8PaufP1DOVQPIcvrsDj
-ZI1/yb0zBKTYKPaRGoTrM2lgX3BKVApSAq1fL8NYzBx9vR3Fwo/5sRPRWl9BpQNx
-v6zDYV2CtrfK9D1opxZulTi45r0aPcVqY1tY79ZdXoO+0dVVJSnEWH3RmCDKS229
-Sg6wTZp+qIHw8WTkAXxX7Wg7wHp5l7Ml7gnoM00pzEvk1ZwPnzyXai0R3ADhhmJx
-3nDUNx+3QmQ+miFD1brn1W+oUf//Wylx2zP5numWVrXuwvPoHUezdJwHXpIAYxBy
-G8RhER/OB7J45lFgwQPEYpkATP3LMZNc19K6jqp70HTgW+3ICTWCIX+5zyOGDLPJ
-nxmmGM5pNuy/s5k4Ojf7+k39fxGwUEZvQ2otlj7lKddBYwjcyQegQ9d25ubiHPGa
-FlAJG1PdUPumulsPto3Kf74d88SXDIjXq9DemVETlUk/hCB0ohNxr3tKFNNS0rOd
-5VPzqPowbIszJmnPQ7FWNFm8BVNE7bIMq0e3KFhfqB/SgCsCAwEAATANBgkqhkiG
-9w0BAQUFAAOCAgEAp3xwZXaNh0tfik23CJ41joRp4Nr8Ud5qaGTZlwseHrxl6Fhr
-eDI2BN4MrFC+IHcF7D98F/sYowWSO0zibp8YZL4IlN5sEFe8XV5jBuyptijinyHX
-ZZ7gMGpAb1F/9OdiLFaQrCsxAg4GyMVWUX1cpptI+jd3cSUZOQ/kHXZ3ls2NmFYk
-dTWPYt/6zTLvDEL2yCdG8qGCnTGmMnykt00F85/vEnWrz5Gpcwg1vJMDlXfNrice
-6IRLE5LI3UZX+8gkTOnz4SSL0Ih97nfXXqDzWXhSvGSeCQ5zW4LCbYF1CZG6wjUE
-tgktNELrDECRtimxozXX9/7hNWQWiCPdddBEBOgfXer0SW2B26BHVsh2PSYeReSj
-6OOXao15BLjLORScn0k5mqpoqUDULpg/MOLPQjq3ZZJMcbzhOBRJ4YhQ8bZ73Amz
-u+MugAve3CqHXFXs5ygCXSIrgcPdYWIKUe/QJOkFyZHA0W3A6fmvpNh8Brm38L8M
-N0//cSZHlJHuFmykDXBqNQPDdUPYMiT3dkVRm7Az+enMpnRs4yfeSSlCyOPJqh7b
-Yrxe6C9CQF5DFatrpF5/bvvxqV/jiwdXr415ikdwHf+LmrORHw3y5dr42++5Cpmm
-wn1T6rOBZEfA8szd+ccp1B3J1VYeEOL3faOTeBni6uDpRiLlxOeed+gDy80=
+MIIDazCCAlOgAwIBAgIUcH3h3SEo76B9rvM0TN8BigTmhswwDQYJKoZIhvcNAQEL
+BQAwRTELMAkGA1UEBhMCQVUxEzARBgNVBAgMClNvbWUtU3RhdGUxITAfBgNVBAoM
+GEludGVybmV0IFdpZGdpdHMgUHR5IEx0ZDAeFw0xOTAyMjUxNzQ2MDdaFw0xOTAz
+MjcxNzQ2MDdaMEUxCzAJBgNVBAYTAkFVMRMwEQYDVQQIDApTb21lLVN0YXRlMSEw
+HwYDVQQKDBhJbnRlcm5ldCBXaWRnaXRzIFB0eSBMdGQwggEiMA0GCSqGSIb3DQEB
+AQUAA4IBDwAwggEKAoIBAQCd3M8rihBCV9soUdRjbhIl32y2+O9Gufxi4slIiANa
+yDaIHJDSqpxgrGGV1IZpyr4Whyq77x8zsSg0cFSZemqeHVJVr9vQZTcIl+o84ktM
+Vtv8CShLgNzf0iwa+2q9t7/9w91c9PG2tWS7bOGC/6RRprxIj8icQe6M9Sh3jWBR
+5VTzlb1Xczf6WMq+9OPJf+xGdq67SaMdWxd96NQ+F2ZAbGB42agZWlvsdIgY/Pll
+fSjkccUteiUk6y/npbtyel1rgW/WSqzwmjgKKOEUGvfyzXZiv3f93ZGesxUppfeu
+h3cQSvBLMfDx3VQY82amX/9uOEUnEtmMeYqbCInQUSszAgMBAAGjUzBRMB0GA1Ud
+DgQWBBTDgysFIBMd5LgLqf8sHjasxNw/lTAfBgNVHSMEGDAWgBTDgysFIBMd5LgL
+qf8sHjasxNw/lTAPBgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQB+
+JTBV4Z9Sox0ebzux9lv8ehdr4jdP87l3n6Wz4VJylTjFi/WIxQ5JElWdvveXv2R5
+YM8BnVB6Xl5Ru+M1szP7F88Q6ywhQxP5xWrrTToaSZXOy2uw8pwEv149ytntmjrC
+T74hFZgXr9lpws+aYQWIPAwfgoHigJqFP53k/Q2jKOdGTwgZKCeKPfnn11podT1f
+k9EeCCOMiWiapbtpC8NVSSVaVVuAXe064GsaaMEqhANnwA17vrQk/fY0mMcU197V
+LdEws20j2W1w6LpCLe8ssBpiVGIumpgdINjmCnjBInaO1xtHFPA+qSy3O/ggz1vX
+AgVXneP8V2eB3lEFhvo8
 -----END CERTIFICATE-----
